packages/arb-rpc-node/txdb/txdb.go

```diff
@@ -209,9 +217,9 @@ func (db *TxDB) DeleteLogs(avmLogs []value.Value) error {
 		}
 
 		for i := oldHeight; i > reorgBlockHeight; i-- {
-			db.snapshotCache.Remove(i)
+			db.snapshotLRUCache.Remove(i)
 		}
-		db.snapshotCache.Remove(reorgBlockHeight)
+		db.snapshotLRUCache.Remove(reorgBlockHeight)
```

Comment:
I think this is an issue if `cacheConfig.LRUSize == 0` so `db.snapshotLRUCache` is nil

Reply:
Similar issue in other places where `snapshotLRUCache` is read. One easy solve would be to error if the size is zero

Reply:
Added checks so snapshotLRUCache is not used if nil.  0 is valid setting when timedcache is added in more-optimizations2 branch